admin-dev/themes/new-theme/js/components/form/form-field-disabler.ts

```diff
@@ -153,8 +161,14 @@ export default class FormFieldDisabler {
     }
 
     elementsToToggle.forEach((elementToToggle: Element) => {
-      elementToToggle.classList.toggle('disabled', disable);
-      elementToToggle.toggleAttribute('disabled', disable);
+      const toggleAvailability = this.params.toggleType === toggleType.availability;
```

Comment:
```suggestion
      const toggleByDisabling = this.params.toggleType === toggleType.availability;
```

Reply:
I guess its also optional depending if i change the option enum namings?